lib/HeroBonus.h

```diff
@@ -216,7 +216,8 @@ class CSelector : std::function<bool(const Bonus*)>
 	BONUS_NAME(SPOILS_OF_WAR) /*val * 10^-6 * gained exp resources of subtype will be given to hero after battle*/\
 	BONUS_NAME(BLOCK)\
 	BONUS_NAME(DISGUISED) /* subtype - spell level */\
-	BONUS_NAME(VISIONS) /* subtype - spell level */
+	BONUS_NAME(VISIONS) /* subtype - spell level */\
+	BONUS_NAME(NO_TERRAIN_PENALTY) /* subtype - terrain type */\
 	

```

Comment:
there should be something (f.e. /* end of list */) on this line as you concatenating it with previous